index.js

```diff
@@ -11,6 +11,14 @@ var onExit = require('signal-exit')
 var stripBom = require('strip-bom')
 var SourceMapCache = require('./lib/source-map-cache')
 var resolveFrom = require('resolve-from')
+var crypto = require('crypto')
+
+function md5 (str) {
+  return crypto
+    .createHash('md5')
+    .update(str, 'utf8')
+    .digest('hex')
```

Comment:
https://blog.risingstack.com/automatic-cache-busting-for-your-css/

> For this, since we care mostly about the hash value changing if images we provide have changed, let’s use MD5, a cryptographic hash function. While MD5 is not appropriate for verification of untrusted data, it does provide uniform distribution when truncated and if we use the first 32 bits, there will be a 1 in 3,506,097 chance of a collision if we have 50 revisions of the same file

via 

https://github.com/sindresorhus/rev-hash